sklearn/metrics/classification.py

```diff
@@ -1009,12 +1167,12 @@ def precision_recall_fscore_support(y_true, y_pred, beta=1.0, labels=None,
 
     It is possible to compute per-label precisions, recalls, F1-scores and
     supports instead of averaging:
+
     >>> precision_recall_fscore_support(y_true, y_pred, average=None,
     ... labels=['pig', 'dog', 'cat'])
     ... # doctest: +ELLIPSIS,+NORMALIZE_WHITESPACE
-    (array([ 0. ,  0. ,  0.66...]),
-     array([ 0.,  0.,  1.]),
-     array([ 0. ,  0. ,  0.8]),
+    (array([0.        , 0.        , 0.66...]),
```

Comment:
What's wrong with the existing formatting? I think it's clearer.

Reply:
I think someone else changed it and it's already in the master branch

Reply:
It does seem to be... but then why does it look like this in the diff? Is the most recent master merged into your branch?

Reply:
I just pulled this file

Reply:
I would recommend against that. Get used to merging master in. It shouldn't be a big deal, but it avoids weird artefacts like this one.

Reply:
I will do that, thanks for the advice!